integration/test_flow_integration.py

```diff
@@ -928,7 +980,20 @@ def test_writing_timedelta_key(setup_teardown_test):
 
 
 def test_write_two_keys_to_v3io_from_df(setup_teardown_test):
-    table = Table(setup_teardown_test.table_name, setup_teardown_test.driver())
+    if setup_teardown_test.driver_name == "SQLDriver":
+        create_sql_table(
+            {"first_name": str, "last_name": str, "city": str},
+            setup_teardown_test._sql_table_name,
+            setup_teardown_test.sql_db_path,
+            ["first_name", "last_name"],
+        )
+        table = Table(
+            setup_teardown_test.table_name,
+            setup_teardown_test.driver(primary_key=["first_name", "last_name"], IsAggregationlessDriver=True),
```

Comment:
move outside the "if" clause and set  IsAggregationlessDriver based on driver_name